services/frontend-service/src/ui/components/CommitInfo/CommitInfo.test.tsx

```diff
@@ -50,4 +60,6 @@ Commit message body line 2`,
     expect(screen.getAllByRole('row', { name: /Commit message body line 1/ })).toHaveLength(1);
     expect(screen.getAllByRole('row', { name: /Commit message body line 2/ })).toHaveLength(1);
     expect(screen.getAllByRole('row', { name: /google, windows/ })).toHaveLength(1);
+    expect(screen.getAllByRole('row', { name: /2024-02-09T09:46:00/ })).toHaveLength(1);
+    expect(screen.getAllByRole('row', { name: /received data about this commit for the first time/ })).toHaveLength(1);
```

Comment:
I think there should be a test that checks the list of environmentNames.

Reply:
Good point, done